Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker:17.05.0-ce-git
+FROM docker:19.03-git
 MAINTAINER Christophe Furmaniak <christophe.furmaniak@gmail.com>
```

Comment:
Should we remove/update this? How do we generally handle things like this?

Reply:
Attribute in comment and update to a remind one?

Reply:
I'd assumed this was an early remind engineer; did we get this code from somewhere else? If so, then we should probably make sure our licenses are all set up properly.

Reply:
git blame doesn't give any clues. Either removal or changing to aws-dev@ would be best

Reply:
I removed it, lmk if I should set it to something else.